recipes/libcublas/meta.yaml

```
@@ -0,0 +1,171 @@
+  - name: libcublas
+    files:
+      - lib/lib*blas*.so.*                            # [linux]
+      - targets/{{ target_name }}/lib/lib*blas*.so.*  # [linux]
+      - Library\bin\cublas*.dll                       # [win]
+      - Library\bin\nvblas*.dll                       # [win]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - sysroot_{{ target_platform }} 2.17    # [linux]
+      host:
+         - cuda-version {{ cuda_version }}
+      run:
+         - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+    test:
+      commands:
+        - test -L $PREFIX/lib/libcublas.so.{{ version }}                                          # [linux]
+        - test -L $PREFIX/lib/libcublas.so.{{ version.split(".")[0] }}                            # [linux]
+        - test -L $PREFIX/targets/{{ target_name }}/lib/libcublas.so.{{ version }}                # [linux]
+        - test -f $PREFIX/targets/{{ target_name }}/lib/libcublas.so.{{ version.split(".")[0] }}  # [linux]
+        - if not exist %LIBRARY_BIN%\cublas64_{{ version.split(".")[0] }}.dll exit 1              # [win]
+        - if not exist %LIBRARY_BIN%\cublasLt64_{{ version.split(".")[0] }}.dll exit 1            # [win]
+        - if not exist %LIBRARY_BIN%\nvblas64_{{ version.split(".")[0] }}.dll exit 1              # [win]
+    about:
+      home: https://developer.nvidia.com/cublas
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_file: LICENSE
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: An implementation of BLAS (Basic Linear Algebra Subprograms) on top of the NVIDIA CUDA runtime.
+      description: |
+        The cuBLAS Library provides a GPU-accelerated implementation of the basic linear algebra subroutines (BLAS).
+      doc_url: https://docs.nvidia.com/cuda/cublas/
+
+  - name: libcublas-dev
+    build:
+      run_exports:
+        - {{ pin_subpackage("libcublas", max_pin="x") }}
+    files:
+      - lib/lib*blas*.so                            # [linux]
+      - lib/pkgconfig                               # [linux]
+      - targets/{{ target_name }}/include           # [linux]
+      - targets/{{ target_name }}/lib/lib*blas*.so  # [linux]
+      - targets/{{ target_name }}/lib/stubs         # [linux]
+      # Not shipping Fortran bindings (in src) on Windows for now
+      - targets/{{ target_name }}/libcublas/src     # [linux]
+      - Library\include                             # [win]
+      - Library\lib                                 # [win]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - sysroot_{{ target_platform }} 2.17    # [linux]
+      host:
+        - cuda-version {{ cuda_version }}
```

Think we may need `libcublas` in `host` to resolve the links in the `libcublas-dev` package

```suggestion
        - cuda-version {{ cuda_version }}
        - libcublas {{ version }}
```

---

I don't think any action is needed here. The packages appear to be correct without this.